docs/src/pages/components/tabs/examples/StyleProps.tsx

```diff
@@ -0,0 +1,20 @@
+import { Tabs, TabItem } from '@aws-amplify/ui-react';
+
+export const StyleProps = () => (
+  <Tabs backgroundColor="var(--amplify-colors-background-primary)">
```

Comment:
Instead of showing how to use the CSS variables again, I think we'd like to demonstrate how to use the [`useTheme` hook](https://ui.docs.amplify.aws/theming#:~:text=Step%202%3A%20Use%20the%20theme%20to%20style%20components) here.

Reply:
Oh, good point. Updated in most recent commits.